cmd/shield/main.go

```diff
@@ -2967,7 +3000,7 @@ func main() {
 			break
 		}
 
-		tbl := tui.NewTable("UUID", "Account", "Created At", "Last Seen", "IP Address", "User Agent")
+		tbl := tui.NewTable("UUID", "Account", "Requested At", "Last Seen", "IP Address", "User Agent")
```

Comment:
One too far; this should remain Created At, because this is auth code / sessions stuff.